test/emqx_shared_sub_SUITE.erl

```diff
@@ -81,7 +83,7 @@ t_random_basic(_) ->
     receive
         {deliver, Topic0, #message{from = ClientId0,
                                    payload = Payload0}} = M->
-        ct:pal("==== received: ~p", [M]),
+        ct:print("==== received: ~p", [M]),
```

Comment:
After finishing debugging we better resort back to `ct:pal` to keep test output clean.